src/I18n.js

```diff
@@ -29,9 +29,12 @@ export default class I18n extends Component {
       this.options.wait = false;
     }
 
+    const { language } = this.i18n;
+    const ready = !!language && this.namespaces.every(ns => this.i18n.hasResourceBundle(language, ns));
```

Comment:
In some of the tests, `i18n.language` was undefined. Under what real-world circumstances is that possible? Does this logic with `!!language` look ok?

Reply:
hm...language should only be empty if not init with resources while initialising or when unable to detect lng

Reply:
I'm guessing it's an idiosyncrasy of the tests/mocking, but if `language` is null, `hasResourceBundle` throws an error, so I'll keep this check 👍